packages/fxa-event-broker/.prettierrc

```diff
@@ -0,0 +1 @@
+{ "useTabs": false, "printWidth": 80, "tabWidth": 2, "singleQuote": true, "trailingComma": "none", "jsxBracketSameLine": false, "parser": "typescript" }
```

Comment:
is it possible to format this (with new lines for prettier rules?)?

Reply:
Yup!